salesforce/10.10/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -9,88 +9,90 @@ You can use Anypoint Connector for Salesforce (Salesforce Connector) as an inbou
 
 * The object store included in Mule for on-premises deployments of Mule apps has no transaction limits.
 * The free version of Object Store for CloudHub deployments of Mule apps has a limit of 10 transactions per second.
 
-For more information about object store versions, see https://docs.mulesoft.com/object-store/#object-store-notes[Object Store Notes].
-
+For more information about Object Store versions, see https://docs.mulesoft.com/object-store/#object-store-notes[Object Store Notes].
 
 === Replay Topic and Replay Channel Listener Operations
 
-The *Replay topic* and *Replay channel listener* operations have the option to continue from the last replay ID they received before restarting the application.
+The *Replay topic* and *Replay channel listener* operations can continue from the last replay ID they received before restarting the application.
 
-When a Mule app starts for the first time, the connector creates an object store that is used to save data related to messages that were processed successfully or that failed:
+When a Mule app starts for the first time, the connector creates an object store that saves data:
 
-* For each message that was processed successfully, the information stored consists of a replay ID associated with the message.
-* For each message that failed, the information stored consists of a number that represents the lowest replay ID for which the processing failed.
+* For each message that processes successfully, the connector stores the replay ID associated with the message.
+* For each message that fails, the connector stores a number that represents the lowest replay ID for which the processing failed.
 
-These two structures ensure that no message is processed twice and that when the application is restarted, it can reprocess the failed messages.
+These two structures ensure that no message is processed twice and that when the application restarts, it can reprocess the failed messages.
 
-For each message that comes through a topic or streaming channel to which the connector is subscribed, the connector updates the object store information using up to four transactions.
+The IDs are stored in the object store for 72 hours, after which they are automatically removed.
 
-[NOTE]
-====
-If you have multiple workers deployed in CloudHub, all Mule instances process each message when using the *Replay Topic* or *Replay Channel* operations.
+For each message that comes through a topic or streaming channel to which the connector is subscribed, the connector updates the object store information using up to four transactions.
 
-To avoid multiple Mule instances processing the same message, you can do one of the following:
+If you have multiple workers deployed in CloudHub, all Mule instances process each message when using the *Replay Topic* operation. To avoid multiple Mule instances processing the same message, you can do one of the following:
```

Comment:
To what does "when using" the operation refer: workers, CloudHub, instances, or message? What is using the operation? Can you reorder the wording to clarify?